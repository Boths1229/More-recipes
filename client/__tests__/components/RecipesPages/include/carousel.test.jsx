import React from 'react';
import { shallow } from 'enzyme';
import Carousel from '../../../../components/Include/carousel';

describe('Carousel component', () => {
  it('matches snapshot', () => {
    const output = shallow(<Carousel />);
    expect(output).toMatchSnapshot();
  });
});
