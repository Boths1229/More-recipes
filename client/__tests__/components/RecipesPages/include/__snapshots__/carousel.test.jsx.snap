// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel component matches snapshot 1`] = `
<div>
  <div
    className="carousel slide"
    data-ride="carousel"
    id="carouselExampleIndicators"
  >
    <ol
      className="carousel-indicators"
    >
      <li
        className="active"
        data-slide-to="0"
        data-target="#carouselExampleIndicators"
      />
      <li
        data-slide-to="1"
        data-target="#carouselExampleIndicators"
      />
      <li
        data-slide-to="2"
        data-target="#carouselExampleIndicators"
      />
    </ol>
    <div
      className="carousel-inner"
    >
      <div
        className="carousel-item active"
      >
        <img
          alt="First slide"
          className="d-block w-100 img-fluid"
          src="test-file-stub"
        />
        <div
          className="carousel-caption d-none d-md-block"
        >
          <h3>
            More-Recipes is a platform for users to share the awesome and exciting recipe ideas they have invented or learnt
          </h3>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="Second slide"
          className="d-block w-100 img-fluid"
          src="test-file-stub"
        />
        <div
          className="carousel-caption d-none d-md-block"
        >
          <h3>
            More-Recipes is a platform for users to share the awesome and exciting recipe ideas they have invented or learnt
          </h3>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="Third slide"
          className="d-block w-100 img-fluid"
          src="test-file-stub"
        />
        <div
          className="carousel-caption d-none d-md-block"
        >
          <h3>
            More-Recipes is a platform for users to share the awesome and exciting recipe ideas t hey have invented or learnt
          </h3>
        </div>
      </div>
    </div>
    <a
      className="carousel-control-prev"
      data-slide="prev"
      href="#carouselExampleIndicators"
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-prev-icon"
      />
      <span
        className="sr-only"
      >
        Previous
      </span>
    </a>
    <a
      className="carousel-control-next"
      data-slide="next"
      href="#carouselExampleIndicators"
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-next-icon"
      />
      <span
        className="sr-only"
      >
        Next
      </span>
    </a>
  </div>
</div>
`;
